addons/account_peppol/models/res_config_settings.py

```diff
@@ -53,6 +53,16 @@ class ResConfigSettings(models.TransientModel):
         help='Register as a PEPPOL user',
     )
 
+    # -------------------------------------------------------------------------
+    # CONSTRAINTS
+    # -------------------------------------------------------------------------
+
+    @api.constrains('account_peppol_endpoint')
+    def _check_account_peppol_endpoint(self):
+        for company in self:
+            endpoint = company.account_peppol_endpoint or ''
+            company.account_peppol_endpoint = ''.join([char for char in endpoint if char.isalnum()])
```

Comment:
wtf? Your constraints is 1) not checking anything and 2) is assigning a field???

Reply:
:sob: